secret_hitler/app.py

```diff
@@ -1,8 +1,31 @@
 import os
 import discord
-from discord.ext import commands
+import logging
 from dotenv import load_dotenv
 from secret_hitler.game import Game, GameStates
+from discord.ext import commands
+from secret_hitler.game import Game, GameStates, Player
+from secret_hitler import config
+
+logFormatter = logging.Formatter('[%(asctime)s][%(levelname)s][%(name)s] %(message)s')
```

Comment:
We don't need to do it in this PR, but we should probably decide on if we are using single or double quotes for strings in this project. Right now we're using both (which inevitably happens when I use Python...), but it would be nice to be tidy and stick to one.

Reply:
opened new issue for this